Loading image into kind cluster incorrectly

In the makefile, it looks like the images are loaded into the kind cluster in an older fashion which didn't work for me.

There is now a `kind load docker-image` command. I'll put up a PR to use this updated logic.

As it stands, the command fails as:

```
docker save aquasec/kube-bench:3964377 -o kube-bench.tar.gz; \
	docker cp kube-bench.tar.gz kube-bench-control-plane:/kube-bench.tar.gz; \
	docker exec kube-bench-control-plane docker load -i /kube-bench.tar.gz;
OCI runtime exec failed: exec failed: container_linux.go:345: starting container process caused "exec: \"docker\": executable file not found in $PATH": unknown
make: *** [kind-push] Error 126
```

--- a/makefile
+++ b/makefile
@@ -48,10 +48,7 @@ endif
 
 # pushses the current dev version to the kind cluster.
 kind-push:
-	docker save $(IMAGE_NAME) -o kube-bench.tar.gz; \
-	docker cp kube-bench.tar.gz $(KIND_CONTAINER_NAME):/kube-bench.tar.gz; \
-	docker exec $(KIND_CONTAINER_NAME) docker load -i /kube-bench.tar.gz;
-	-rm -f kube-bench.tar.gz
+	kind load docker-image $(IMAGE_NAME) --name $(KIND_PROFILE)
 
 # runs the current version on kind using a job and follow logs
 kind-run: KUBECONFIG = "$(shell kind get kubeconfig-path --name="$(KIND_PROFILE)")"

